src/Llvm.NET/Native/Generated.cs

```diff
@@ -848,7 +848,7 @@ internal static partial class NativeMethods
         internal static extern LLVMTypeRef LLVMDoubleTypeInContext( LLVMContextRef @C );
 
         [DllImport( LibraryPath, EntryPoint = "LLVMX86FP80TypeInContext", CallingConvention = CallingConvention.Cdecl )]
-        internal static extern LLVMTypeRef LLVMX8FP80TypeInContext( LLVMContextRef @C );
+        internal static extern LLVMTypeRef LLVMX86FP80TypeInContext( LLVMContextRef @C );
```

Comment:
Please split these changes out to a separate PR so they can be merged without the other changes